### ISSUE TITLE
RUBYGEMS_GEMDEPS and rbx

### ISSUE THREAD

**Issue by lloeki** [postmodern/ruby-install#199]
Having `RUBYGEMS_GEMDEPS` set to `-` makes `ruby-install rbx` totally blow up. It might be worth forcefully unsetting it.

**Comment by postmodern** [comment]
Should we also unset GEM_PATH for good measure?

**Comment by lloeki** [comment]
Makes sense. Also, some distros (Arch Linux) have the habit of having a `/etc/gemrc` (or something like that) that may trip some things up by setting a `--user` flag (IIRC) for the `gem` command to use.

**Comment by postmodern** [comment]
Closing as support for `rbx` has been dropped a while ago due to inactivity.
